### DISCUSSION TITLE
Login with YubiKey without master password

### DISCUSSION THREAD

**Discussion by Danielbcvv** [dani-garcia/vaultwarden#5098] [Q&A]
I looked around here, but I couldn't find anything conclusive/that I understand. Is it possible to log into the password vault without the master password using a YubiKey? If so, how? Apologies if this has already been answered in another discussion.

**Comment by BlackDex** [comment]
No it is not. Currently you can only use a Yubikey for 2FA

**Comment by Danielbcvv** [comment]
Thank you for letting me know! I found an alternative way to make it easier; I set the long touch as the master password.
